config.json

```diff
@@ -8,23 +8,73 @@
     "representer": false,
     "analyzer": false
   },
-  "blurb": "TODO: add blurb",
+  "blurb": "Simple, expressive, and portable, Arturo is a batteries-included language inspired by REBOL and Smalltalk.",
   "version": 3,
   "online_editor": {
     "indent_style": "space",
     "indent_size": 4
   },
-  "files": {
-    "solution": [],
-    "test": [],
-    "example": [],
-    "exemplar": []
+  "test_runner": {
+    "average_run_time": 10
   },
-  "exercises": {
-    "concept": [],
-    "practice": []
+  "files": {
+    "solution": [
+      "src/%{kebab_slug}.art"
+    ],
+    "test": [
+      "tests/test-%{kebab_slug}.art",
+      "tester.art"
```

Comment:
@BNAndras I wonder if this should be an invalidator file. If we include it here, it will be shown in the tests in the online editor, which doesn't provide much value for the user I think.

Reply:
Makes sense.